Assets/Scripts/UI/TextBox.cs

```diff
@@ -0,0 +1,50 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+public class TextBox : MonoBehaviour {
+
+	Text _UIText = null;
+
+	[SerializeField] KeyCode _clearKey = 0;
+
+	void Start ()
+	{
+		_UIText = GetComponent<Text>();
+	}
```

Comment:
Could probably just inherit from Text instead of holding a reference to another component.